src/ServiceInsight/MessageList/MessageListView.xaml.cs

```diff
@@ -56,7 +56,11 @@ void OnBeforeLayoutRefresh(object sender, CancelRoutedEventArgs e)
 
         void SortData(ColumnBase column, ListSortDirection order)
         {
-            Model.RefreshMessages(column.Tag as string, order == ListSortDirection.Ascending);
+            var orderBy = column.Tag as string;
+            var ascending = order == ListSortDirection.Ascending;
+
+            Model.ChangeSorting(orderBy, ascending);
+            Model.RefreshMessages();
```

Comment:
Given this is the only call to `ChangeSorting`, i.e. the first method is never called without calling the second, would it be better to encaspulate that behind a `RefreshMessages` overload which take `orderBy` and `ascending` parameters?